_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Button up or button down, you get five **customisable** action buttons to control playback. Batteries not included. There is a new <code><img src="/img/icons/play_arrow-24dp.svg" /> notification</code> category in the app settings, from where you can pick actions and the buttons to carry out your bidding. Play/pause, forward/rewind, previous/next, shuffle, repeat, close, or even nothing! The decadency.
 
-Then, you can choose which three of these five buttons show in the compact notification (remember that little arrow you use to expand/contract the notification?).
+Three of these five buttons can be shown in the compact notification (remember that little arrow you use to expand/contract the notification?).
 
-For users on **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As we've noted above, MediaStyle notifications adapt to the colours of the cover art, which looks just plain awesome.
+For the betrodden of **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As noted above, MediaStyle notifications adapt to cover art colours. Big.
 
-For users on **Android 10+**: Seekbar! The old notification had a progress bar, but that was just a visual indicator. Now you can actually seek from the notification itself. Marvel at the massive boost in convenience, and have fun seeking!
+For the modern human using **Android 10+**: SeekBar! The old notification had a progress bar. Now you can actually seek from the notification itself. Marvel at the massive boost in convenience, and you shall find what you seek.
 
-For users on **Android 11**: Congratulations! You get a notification now! NewPipe's media session will show up in the notification shade and in Quick Settings just as advertised.
+For the beyond human users of **Android 11**: Congratulations! You get a notification! NewPipe's media session shows up in the notification shade and in "Quick Settings", just as advertised. Enjoy the envy of the world.
 
 <div class="media-row media-items-2">
 <img class="no-flow img-responsive" style="align-self: center" src="/img/screenshots/notification_9.png"/>
 <img class="no-flow vertical" src="/img/screenshots/notification_11.png"/>
 </div>
-This cool contribution was brought to us in its original form by [@cool-student](https://github.com/cool-student). Sadly, we couldn't get in touch with them when we were ready to focus on this PR, so [@wb9688](https://github.com/wb9688) and [@Stypox](https://github.com/Stypox) swooped in to save the day! [@wb9688](https://github.com/wb9688) updated the code to conform to our standards and coding style, and [@Stypox](https://github.com/Stypox) did the (painstaking!) work of rebasing the code after the unified player PR was merged into the development branch.
```

Comment:
Don't use "them" for known single pronoun. That is poor language. Use "the person", use "the author", anything. "their work". OK, standard fare. Neither here nor there.

Reply:
> Don't use "them" for known single pronoun. That is poor language.

No, it is commonly accepted usage for persons of unknown gender. Check online.

> Use "the person", use "the author", anything. 

Using a noun again and again is poor ~~language~~ grammar. That is why we have pronouns. Dude, this is school level stuff.

> "their work"

Where is that? I don't see it.

Reply:
It is used for third person singular unknown in English. You are using it for known singular.